Changelog.md

```diff
@@ -106,12 +106,73 @@ This changelog covers the entire dave project which includes a few repositories,
-    - Demonstration of addition of custom SDF tags for extending model properties and corresponding influence on sensing [PR#164](https://github.com/Field-Robotics-Lab/dave/pull/164)
+- Object model degradation tool
+  - 3D mesh modification via Blender script [PR#160](https://github.com/Field-Robotics-Lab/dave/pull/160)
+  - Parameterization proof of concept for SDF friction values via embedded Ruby script [PR#163](https://github.com/Field-Robotics-Lab/dave/pull/163)
+  - Demonstration of addition of custom SDF tags for extending model properties and corresponding influence on sensing [PR#164](https://github.com/Field-Robotics-Lab/dave/pull/164)
+
+## [4.3.0] - 2022-02-25
+
+### Updated
+
+- Ocean current plugin
+  - Search for database file in all gazebo paths (GAZEBO_MODEL_PATH, GAZEBO_RESOURCE_PATH, etc) [dave PR #190](https://github.com/Field-Robotics-Lab/dave/pull/190)
+
+- Multibeam Sonar plugin
+  - Docker environment using OSRF Rocker and dockwater [Installation using docker](https://github.com/Field-Robotics-Lab/dave/wiki/Multibeam-Forward-Looking-Sonar#option-a-use-docker)
+  - New colorized live sonar image viewer [Multibeam Sonar PR #38](https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar/pull/38)
+
+- Object model degradation tool
+  - Bug fixes; tutorial updates documenting how to add new distortion methods [dave PR #185](https://github.com/Field-Robotics-Lab/dave/pull/185)
+  - Set the amount of vertex randomization offset `VERT_RAND_MAX` based on object size; degraded model examples [dave PR #212](https://github.com/Field-Robotics-Lab/dave/pull/212)
+
+### Added
+
+- Multibeam Sonar Plugin
+  - GPURay-based multibeam sonar with fidelity flag for number of elevation rays [Multibeam Sonar PR #37](https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar/pull/37)
+  - Custom SDF tags of model description for multibeam sonar reflecitivty [Reflectivity by custom sdf tags](https://github.com/Field-Robotics-Lab/dave/wiki/Multibeam-Forward-Looking-Sonar#reflectivity-by-custom-sdf-tags)
+  - Multibeam Sonar URDF for standalone and robot description [Multibeam Sonar PR #38](https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar/pull/38)
+   - Documentation and tutorial for the local search scenario demonstration with GPURay-based multibeam sonar [wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Multibeam-Forward-Looking-Sonar#degradaded-object-detection-scenarios)
+   - Documentation and tutorial for the degraded object detection scenarios with distorted mesh models [wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Multibeam-Forward-Looking-Sonar#degradaded-object-detection-scenarios) and [Multibeam Sonar PR #33](https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar/pull/33)
+   
+- Bathymetry Converter
+  - Docker image distribution including neccesary libraries and excutables [Docker Hub image](https://hub.docker.com/repository/docker/woensugchoi/bathymetry_converter)
```

Comment:
Use the public page. The `repository` URL requires logging into Docker Hub to see the page.

```suggestion
  - Docker image distribution including neccesary libraries and excutables [Docker Hub image](https://hub.docker.com/r/woensugchoi/bathymetry_converter)
```